install.md

```diff
@@ -611,11 +680,16 @@ sudo yum update cri-o
 
 If updating to a patch version (for example, ``VERSION=1.8.3``
   ), run
-```
+
+```shell
 apt update cri-o cri-o-runc
 ```
-Otherwise, be sure that the environment variable ```$OS``` is set to your operating system as the appropriate field in the following table
-To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
+<!-- markdownlint-enable MD024 -->
+
+Otherwise, be sure that the environment variable ```$OS``` is set to your operating
```

Comment:
```suggestion
Otherwise, be sure that the environment variable ```$OS``` is set to the appropriate value from the following table for your operating system.
```